internal/rootcmd/root.go

```diff
@@ -0,0 +1,37 @@
+package rootcmd
+
+import (
+	"context"
+	"flag"
+
+	"github.com/artefactual-sdps/enduro/internal/config"
+	"github.com/peterbourgon/ff/v3/ffcli"
+)
+
+type Config struct {
+	config.Configuration
+	Verbose bool
+}
+
+func New() (*ffcli.Command, *Config) {
+	var cfg Config
+
+	fs := flag.NewFlagSet("enduro-ctl", flag.ExitOnError)
+	cfg.RegisterFlags(fs)
+
+	return &ffcli.Command{
+		Name:       "enduro-ctl",
+		ShortUsage: "enduro-ctl [flags] <subcommand> [flags] [<arg>...]",
+		FlagSet:    fs,
+		Exec:       cfg.Exec,
+	}, &cfg
+}
```

Comment:
Are you using this constructor? Looks unused at the moment.

Reply:
Yup it is unused